### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
**About my pull request**

In this Branch I:

- Added a desktop version to my project.
- Used new pictures for the desktop version.
- Modified HTML file.
- Modified CSS file.

### CODE REVIEW

**Comment by mukhammadrizooff** [comment]
I did this project with interest.🧐

File: index.html [hunk 1]
```diff
@@ -197,81 +200,65 @@ <h3 class="works__project-title">Multi-Post Stories</h3>
...
+                </li>
+                <li class="lang-tags">
+                  <img src="images/css.png" alt="css logo" />CSS
+                </li>
+              </ul>
+            </li>
+            <li><h3 class="about-myself__list-title">Frameworks</h3></li>
+            <li><h3 class="about-myself__list-title">Skills</h3></li>
+          </ul>
+        </section>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] AWESOME job 👍, everything is working as expected but there is one thing left to be done. Inside about me section after your list titles there should be arrow icon. I'm sure you can fix it easily 😄🍀

**Your design** 

image 

**Figma design**

image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @mukhammadrizooff , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mukhammadrizooff) [reply index.html]: Yes, I can thank you.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @mukhammadrizooff , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**